### DISCUSSION TITLE
DNS Resolver

### DISCUSSION THREAD

**Discussion by inam4028** [netbox-community/netbox#10637] [General]
Hello, 

Does anyone have some examples of using Netbox IPAM/ipaddress webhooks to make record changes in PowerDNS via its API that they would be willing to share? I've been exploring both tools to help us implement IPAM and host an internal DNS subdomain for all of our Linux systems.

OR any recommendations of using a DNS server with netbox integration? 

Thank you

**Comment by candlerb** [comment]
I wrote a receiver to translate a webhook into DDNS updates (RFC2136), talking to Bind. You could use it as a starting point if you want to integrate with the PDNS API instead - or you could use it as-is with PDNS's own [RFC2136 support](https://doc.powerdns.com/authoritative/dnsupdate.html).

https://github.com/candlerb/netbox-webhook-dnsupdate
